aws/resource_aws_instance_test.go

```diff
@@ -3280,6 +3280,19 @@ data "aws_availability_zones" "available" {
 `, strings.Join(excludeZoneIds, "\", \""))
 }
 
+func testAccAvailableAZsNoOptInConfig() string {
```

Comment:
nice! we could also document this function in `https://github.com/terraform-providers/terraform-provider-aws/blob/master/docs/contributing/running-and-writing-acceptance-tests.md` for contributors to re-use

Reply:
I added docs on this!